### PR TITLE
List devtools

### PR DESCRIPTION
**Pull request by uF4No**
# What :computer: 
* Adds a list of popular dev tools

# Why :hand:
* We need to list all dev tools compatible with Era

# Evidence :camera:
In preview

# Notes :memo:
This is a WIP PR

### CODE REVIEW

File: docs/.vuepress/config.ts [hunk 1]
```diff
@@ -5,6 +5,10 @@ import theme from "./theme.js";
 import { pwaPlugin } from "vuepress-plugin-pwa2";
 import { getDirname, path } from "@vuepress/utils";
 
+// import registerPkg from "@vuepress/plugin-register-components";
```
Comment (MexicanAce) [inline docs/.vuepress/config.ts:8]: Can we remove these comments please?

File: docs/.vuepress/public/images/tools/herodotus-logo.png
Comment (MexicanAce) [inline docs/.vuepress/public/images/tools/herodotus-logo.png:1]: Do we need both JPEG and PNG formats?

File: docs/assets/data/devtools.json [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+          "name": "In-memory node",
+          "category": "Testing",
+          "description": "Super fast node built in Rust for local testing and debugging.",
+          "logo": "memory-node-logo.png",
+          "url": "https://era.zksync.io/docs/tools/testing/era-test-node.html"
+        },
+        {
+          "name": "Docker setup",
+          "category": "Testing",
+          "description": "Simulate your own tesnet locally running an L1 and L2 nodes.",
```
Comment (MexicanAce) [inline docs/assets/data/devtools.json:117]: typo: 
```suggestion
          "description": "Simulate your own testnet locally running an L1 and L2 nodes.",
```

File: docs/.vuepress/config.ts (hunk 1 above)
Comment (uF4No) [reply docs/.vuepress/config.ts]: fixed in https://github.com/matter-labs/zksync-web-era-docs/pull/657/commits/fef65a21bb07e38ff0d91c7b2fda556f3ea50ae9

File: docs/.vuepress/public/images/tools/herodotus-logo.png
Comment (uF4No) [reply docs/.vuepress/public/images/tools/herodotus-logo.png:1]: fixed in https://github.com/matter-labs/zksync-web-era-docs/pull/657/commits/b734a302ab1b90db2bba116776b09634ed2a7afe

File: docs/.vuepress/public/images/tools/chainstack-logo.png
[the remark this answers is not in the document]
Comment (uF4No) [reply docs/.vuepress/public/images/tools/chainstack-logo.png:1]: fixed in https://github.com/matter-labs/zksync-web-era-docs/pull/657/commits/b734a302ab1b90db2bba116776b09634ed2a7afe

**Review by MexicanAce — APPROVED** [review]
LGTM

**Pull request merged**